config/rbac/role.yaml

```diff
@@ -6,6 +6,18 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
+- apiGroups:
```

Comment:
This allows the operator's service account to create configmaps in any namespaces. Could it be restricted to dbaas namespace only (ie, create a separate role and grant RBAC for the dbass namespace only)?

Reply:
the config map is tied with the DBaaS operator namespace, see here, https://github.com/redhatHameed/crunchy-bridge-operator/blob/registration-cm/controllers/dbaas.redhat.com/configmap.go#L39 
kubebuilder auto-generated RBAC roles/role bindings don't have an option  to limit to a specific namespace, When I tried to change the ClusterRole/ClusterRoleBinding to Role and RoleBinding(namespace), it kept complaining that the ServiceAccount has no permissions to list configmap etc

Reply:
Yes it seems a limitation of kubebuilder.  If we have to use cluster role here, we could specify the configmap name so that the permission is only in this configmap, not any other configmaps.

Reply:
so with that, it will not work when the operator installed in the namespace scope. so keeping cluster level scope until we find the better solution.